src/App_old.jsx

```diff
@@ -0,0 +1,46 @@
+import 'bulma/css/bulma.css';
```

Comment:
why do you need this file?

Reply:
this file takes the styles for the buttons